test/menuFactory.spec.js

```diff
@@ -413,5 +413,30 @@ describe('menuFactory', () => {
       ReactDOM.render(<Menu isOpen />, container);
       expect(component.state.isOpen).to.be.true;
     });
+
+    it('will not change when isOpen prop was not changed', () => {
+      let container = document.createElement('div');
+      let ParentComponent = React.createClass({
+        getInitialState() {
+          return { collapsed: true };
+        },
+        changeProps() {
+          this.setState({
+            collapsed: !this.state.collapsed
+          });
+        },
+        render() {
+          return (
+            <Menu ref="menu" {...this.state} />
+          );
+        }
+      });
```

Comment:
Actually it should be just 
```jsx
render() {
    return (
      <Menu ref="menu" />
    );
  }
```
My bad, it is confusing with the spread operator, because what I wanted to show there in that test case is that `<Menu />` component (that's inner state's `isOpen` is set to `true`) does not have any relevant props set, but still gets closed when the state of `ParentComponent` changes.

The `collapsed` prop is not relevant to burger menu component (`<Menu />`) but to `<ParentComponent />`